internal/lib/container_server.go

```diff
@@ -37,8 +38,7 @@ const ContainerManagerCRIO = "cri-o"
 // ContainerServer implements the ImageServer
 type ContainerServer struct {
 	runtime              *oci.Runtime
-	store                cstorage.Store
-	storageImageServer   storage.ImageServer
+	storageImageServer   storage.MultiStoreServer
```

Comment:
hm what was the motivation behind calling it storageImageServer? I would imagine it would be something like "stores" or "multiStore"